docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ConvertToGrayScale.cs

```diff
@@ -40,20 +41,32 @@ public static void Example()
                            .Append(mlContext.Transforms.ConvertToGrayscale("Grayscale", "ImageObject"));
 
             var transformedData = pipeline.Fit(data).Transform(data);
-
             // The transformedData IDataView contains the loaded images column, and the grayscaled column.
-            // Preview of the transformedData
+            
+            // Preview the transformedData. 
             var transformedDataPreview = transformedData.Preview();
+            PrintPreview(transformedDataPreview);
+            // ImagePath    Name         ImageObject            Grayscale
+            // tomato.bmp   tomato       System.Drawing.Bitmap  System.Drawing.Bitmap
+            // banana.jpg   banana       System.Drawing.Bitmap  System.Drawing.Bitmap
+            // hotdog.jpg   hotdog       System.Drawing.Bitmap  System.Drawing.Bitmap
+            // tomato.jpg   tomato       System.Drawing.Bitmap  System.Drawing.Bitmap
+        }
 
-            // Preview of the content of the images.tsv file
-            // The actual images, in the Grayscale column are of type System.Drawing.Bitmap.
-            //
-            // ImagePath    Name        ImageObject                   Grayscale
-            // tomato.bmp   tomato      {System.Drawing.Bitmap}     {System.Drawing.Bitmap}
-            // banana.jpg   banana      {System.Drawing.Bitmap}     {System.Drawing.Bitmap}
-            // hotdog.jpg   hotdog      {System.Drawing.Bitmap}     {System.Drawing.Bitmap}
-            // tomato.jpg   tomato      {System.Drawing.Bitmap}     {System.Drawing.Bitmap}
+        private static void PrintPreview(DataDebuggerPreview data)
+        {
+            foreach (var colInfo in data.ColumnView)
+                Console.Write("{0, -25}", colInfo.Column.Name);
 
+            Console.WriteLine();
+            foreach (var row in data.RowView)
+            {
+                foreach (var kvPair in row.Values)
+                {
```

Comment:
>{ [](start = 16, length = 1)

for 1-liner blocks, let's not use {} to keep the samples short. similar to foreach loop on line 58

Reply:
actually these are 2-liners. may use Linq to shorten it.

---
In reply to: [271452481](https://github.com/dotnet/machinelearning/pull/3165#discussion_r271452481) [](ancestors = 271452481)

Reply:
keeping 2 for loops to keep logic simple

---
In reply to: [271453121](https://github.com/dotnet/machinelearning/pull/3165#discussion_r271453121) [](ancestors = 271453121,271452481)